### PR TITLE
Add PIPENV_DONT_LOAD_ENV in advanced docs

### PR DESCRIPTION
**Pull request by rodxavier**

### CODE REVIEW

File: docs/advanced.rst [hunk 1]
```diff
@@ -262,6 +262,9 @@ If your ``.env`` file is located in a different path or has a different name you
 
     $ PIPENV_DOTENV_LOCATION=/path/to/.env pipenv shell
 
+To prevent pipenv from loading the ``.env`` file, set the `PIPENV_DONT_LOAD_ENV` environment variable::
```
Comment (nateprewitt) [inline docs/advanced.rst]: double backticks on PIPENV_DONT_LOAD please!

Comment (rodxavier) [reply docs/advanced.rst]: pushed

**Comment by rodxavier** [comment]
@nateprewitt pushed the change

**Comment by nateprewitt** [comment]
Looks good, thanks @rodxavier!

**Pull request merged**